hepcrawl/spiders/osti_spider.py

```diff
@@ -0,0 +1,197 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of hepcrawl.
+# Copyright (C) 2019 CERN.
+#
+# hepcrawl is a free software; you can redistribute it and/or modify it
+# under the terms of the Revised BSD License; see LICENSE file for
+# more details.
+
+"""Spider for OSTI/SLAC"""
+
+from __future__ import absolute_import, division, print_function
+
+import json
+from datetime import datetime
+
+from furl import furl
+
+import link_header
+
+from scrapy import Request
+
+from .common.lastrunstore_spider import LastRunStoreSpider
+from ..parsers import OSTIParser
+from ..utils import (
+    ParsedItem,
+    strict_kwargs,
+)
+
+
+class OSTISpider(LastRunStoreSpider):
+    """OSTI crawler
+
+    Uses the OSTI REST API v1 https://www.osti.gov/api/v1/docs
+    to harvest records associated with SLAC from OSTI
+
+    Example:
+        Using the OSTI spider::
+
+            $ scrapy crawl OSTI -a "from_date=2019-04-01"
+
+    """
+
+    name = 'OSTI'
+    osti_base_url = 'https://www.osti.gov/api/v1/records'
+
+    @strict_kwargs
+    def __init__(self,
+                 from_date=None,
+                 until_date=None,
+                 rows=400,
+                 product_types=u'Journal Article,Technical Report,Book,Thesis/Dissertation',
```

Comment:
why a comma-separated string and not a tuple here?

Reply:
when I run
`scrapy crawl -a"product_types=..." -a "from_date=01/01/2015" -a "until_date=09/12/2019" OSTI`
it's easiest to pass a comma separated string

using `-a="product_types=Book" -a"product_types=Technical Report"`
does not work, since `-a` doesn't aggregate